app/views/insights/community.html.erb

```diff
@@ -0,0 +1,114 @@
+<div class="container new-discussion-timeline experiment-repo-nav">
+  <div class="repository-content">
+    <div class="columns">
+      <div class="column one-fourth">
+        <nav class="menu" data-pjax>
```

Comment:
I see duplication of ``<nav>`` tag in some files. What about creation of partial named ```_menu.html.erb```?